hpe_3par_kubernetes_manager.py

```diff
@@ -25,6 +25,48 @@
 timeout = 180
 
 
+def hpe_create_service_object(yml):

```

Comment:
Please add detail description for all the methods

Example : def hpe_create_service_object(yml)

hpe_create_service_object - This function creates a Kubernetes service using the provided service configuration (body)
Parameters:
	Required:
		1. (yml): The body of the Kubernetes service configuration.
	Optional:
		None
Returns:
	resp: The response object from the Kubernetes API after service creation.
Raises:
    Exception: If there's an error while creating the service, it catches and raises the exception.

Reply:
Added detailed description for methods, resolving all conversations related to it